config/settings.yml

```diff
@@ -0,0 +1,9 @@
+development:
+  display_output: true
+
+production:
+  display_output: true
+
+test:
+  display_output: false
+
```

Comment:
Either delete this file or turn it into a template that contains all config setting options